e2e/api/steps.js

```diff
@@ -182,7 +186,7 @@ module.exports = {
     console.log('Is PBAv3 toggle on?: ' + pbaV3);
 
     let bodyText = 'Your claim will not be issued until payment is confirmed.';
-    let headerText = '# Please now pay your claim fee\n# using the link below';
```

Comment:
Why we need to modify this header? I think its commited accidently.

Reply:
Yes, true, can't figure out why this was changed. It is in Master, so I'll put it back. Thank you!